frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx

```diff
@@ -36,9 +36,9 @@ const AttachDeploymentToOBCModal = withHandlePromise((props: AttachDeploymentToO
   const { handlePromise, close, cancel, resource, deployments } = props;
 
   const obcName = getName(resource);
-  const deploymentData = _.get(deployments, 'data');
-  const inProgress = _.get(props, 'loaded');
-  const errorMessage = _.get(props, 'loadError');
+  const deploymentData = deployments?.data;
+  const inProgress = props?.[0]?.loaded;
+  const errorMessage = props?.[0]?.loadError;
```

Comment:
Why [0] ?

Reply:
@afreen23  Adding [0] is wrong thing. But props doesn't contain loaded or loadError. It's not taking them.